src/Castle.Windsor/Castle.Windsor.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

Comment:
Castle.Core is targeting `net462;netstandard2.0;netstandard2.1;net6.0`. Is there any value in this also having net6.0?

Reply:
I initially was going to add `net6.0` as part of this issue, but realised it would be better to do them separately. Had a quick throw-away stab at it, and it'll involve some fiddling with package references, but seems pretty straightforward.

Reply:
One at a time sounds great, easier for me to review. Obviously need to get the build set up with it too.